<?xml version="1.0"?>
<launch>
    <node pkg="usb_cam" name="usb_cam" type="usb_cam_node" output="screen">
    </node>

    <node pkg="drone_marker_pkg" name="marker_detect_node" type="rosImage_to_cv2_to_rosImage_node.py" output="screen">
    </node>

    <node pkg="image_view" type="image_view" name="image_view" output="screen">
        <remap from="image" to="/marker_detect/image_raw"/>
    </node>

</launch>
